pages/api/email/[list].ts

```diff
@@ -24,52 +99,30 @@ export default async function handler(
       message: 'Email API: Invalid http method. | Only POST is accepted.',
```

Comment:
Not important be should be `405` for method not allowed

Reply:
TIL. Fixed